src/components/Dialog/SingleSend/index.tsx

```diff
@@ -230,13 +230,13 @@ const SingleSend = () => {
     }
 
     const maxingOut = () => {
+        if (new BigNumber(chosenBalance!.amount!).isLessThan(amountToAcudos(0.5))) {
```

Comment:
lets extract this into a constant, since it is used on several places, and if we need to change it like this there is a chance to miss it.